shared/js/toc.jsx

```diff
@@ -125,27 +128,32 @@ loop.shared.toc = (function(mozL10n) {
               label={this.state.roomName}
               onEditionComplete={this.exitEditMode} />
           </div>
-          <RoomPresenceView />
+          <RoomPresenceView
+            participantStore={this.props.participantStore} />
           <RoomActionsView
             addUrlTile={this.props.addUrlTile} />
         </div>
       );
     }
   });
 
-  // XXX akita: Make this work
   var RoomPresenceView = React.createClass({
-    propTypes: {},
+    propTypes: {
+      participantStore: React.PropTypes.instanceOf(loop.store.ParticipantStore).isRequired
+    },
 
     render: function() {
       return (
         <div className="room-active-users">
-          <div className="room-user" data-name="Pau Masiá">
-            <span>{'P'}</span>
-          </div>
-          <div className="room-user" data-name="Manu">
-            <span>{'M'}</span>
-          </div>
+          {
+            this.props.participantStore.getOnlineParticipants().map(function(participant, index) {
+              return (
+                <div className="room-user" data-name={participant[1].participantName} key={index}>
+                  <span>{participant[1].participantName.substring(0, 1).toUpperCase()}</span>
```

Comment:
Doh. Could you fix up getOnlineParticipants to just return an array of participants instead of array of `[key, value]`? Should be able to do `[...this._storeState.participants.values()]` and update the method documentation with the return value type and update test to check that the online participants returned is a participant with a name?

The lines here would then just be `participant.participantName`